src/pages/workspaces/workspace/Data.js

```diff
@@ -748,7 +748,10 @@ const WorkspaceData = _.flow(
             ])
           }, [h(ButtonOutline, {
             disabled: !canEditWorkspace || uploadingWDSFile,
-            tooltip: Utils.cond([uploadingWDSFile, 'Upload in progress'], canEditWorkspace ? 'Add data to this workspace' : editWorkspaceErrorMessage),
+            tooltip: Utils.cond(
+              [uploadingWDSFile, () => 'Upload in progress'],
+              () => (canEditWorkspace ? 'Add data to this workspace' : editWorkspaceErrorMessage)
+            ),
```

Comment:
both conditions in this `Utils.cond` have actual syntax changes, as well as adding linebreaks.